src/main/resources/db/changelog/changelog-1.0.0.xml

```diff
@@ -10,4 +10,5 @@
     <include file="changes/v1.0.0/2024-08-06-add-awaiting-delivery-step.xml" relativeToChangelogFile="true"/>
     <include file="changes/v1.0.0/2024-08-30-create-mediated_request_workflow_log-table.xml" relativeToChangelogFile="true"/>
     <include file="changes/v1.0.0/2024-09-18-make-some-mediated_request-columns-nullable.xml" relativeToChangelogFile="true"/>
-</databaseChangeLog>
\ No newline at end of file
+    <include file="changes/v1.0.0/2024-11-21-create_fake_patron_link-table.xml" relativeToChangelogFile="true"/>
```

Comment:
I believe it should be v1.1.0 package

Reply:
@roman-barannyk changes are applied to rancher DB so I can't move the files now.